### ISSUE TITLE
feat[dev]: ✨ implement A11y improvements to sd-tab, sd-tab-panel and sd-tab-group

### ISSUE THREAD

**Issue by yoezlem** [solid-design-system/solid#1489]
## Description
This task describes the Storybook implementation for the sd-tab, sd-tab-panel and sd-tab-group component as part of the A11y improvements outlined in Epic https://github.com/solid-design-system/solid/issues/1465.

A11y improvements for sd-tab, sd-tab-panel and sd-tab-group can be found here: [sd-tab, sd-tab-panel and sd-tab-group Notes](https://www.notion.so/matuzo/sd-tab-sd-tab-panel-sd-tab-group-10df3f6e1bc080ae852ef63211e2b5d2).

------------------------------

## Comment
**Problem:** Especially in tab groups with many items, users have to press the Tab key often whenever they want to access the content after picking a tab. (see [Problem #1168](https://github.com/solid-design-system/solid/issues/1168))
**Solution:** Keep the arrow keys for switching between tabs, but only make the active tab focusable via Tab, allowing users to focus content immediately after switching tabs.

**task:**
- [x] switching between tabs with arrow keys (and only with the keys)
- [x] activating a tab with space/enter and tab key -> only for manual activation
- [x] in addition tab key focuses content

**Problem:** The buttons in scrollable tabs are useless for keyboard users because all tabs, no matter whether they're visible, are focusable, which means that as soon as users have reached the "next" button, they've already gone through all buttons in the list.
https://www.notion.so/matuzo/sd-tab-sd-tab-panel-sd-tab-group-10df3f6e1bc080ae852ef63211e2b5d2?pvs=4#10ef3f6e1bc080bdb10de10a31fa5c77 --> Video in the link
**Solution:** Put the "next" button right next to the "prev" button in the DOM and/or make invisible buttons inert.

**task:**
- [x] make buttons not focusable (next/prev)

**Problem:** The “prev”/”next” buttons have no accessible name.
**Solution:** Add a label to the icons.

## Notes
- Disabled tabs: See [@sd-button](https://www.notion.so/sd-button-104f3f6e1bc0800b9f72d2a046316ec3?pvs=24). Everything I wrote there about disabling form elements applies here as well.

## Best Practices
- Hide content in tabs only if it really benefits users.

## Related WCAG criteria
- https://www.w3.org/WAI/WCAG22/quickref/#name-role-value

## Additional resources
- https://www.w3.org/WAI/ARIA/apg/patterns/tabs/

## Template Ticket
- Pick up the template ticket when working on the component story for A11y improvement, as it references the component and will be resolved once the component is made accessible:
#1524 

## DoR
- [x] Item has been estimated by the team
- [x] Item dependencies have been identified and documented

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch
